recipes/misic-napari/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "misic-napari" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/misic-napari-{{ version }}.tar.gz
+  sha256: c61f0778a97af2dba041918268d68349cc000cfc1634ea246f4f1ea9b7eda0c8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - napari
+    - napari-plugin-engine
+    - python >=3.6
+    - tensorflow
```

Comment:
we should probably pin this to a minimum ?

Reply:
```suggestion
    - tensorflow >=2
```